src/DevServer/index.ts

```diff
@@ -296,6 +313,15 @@ export class DevServer {
         this.manifest.generate()
       }
 
+      // TODO: If a .ts file in the resources/js folder is changed,
+      // The whole server is restarted. This is not ideal.
+      // Not sure how to approach this yet. First idea was to
+      // use .adonisrc.metaFile but seems a bit hacky.
+      // This kind of hardcoded condition also seems hacky.
+      if (relativePath.includes('resources')) {
```

Comment:
As explained.

Reply:
( to clarify: this condition is just a temporary and dirty patch. I have 0 intention of merging the PR with this  )

Reply:
What's the goal here?

Reply:
I add a file in resources/js and it's a ".ts" file, then I update it : the dev server is totally reloaded. And when I talk about the dev server, I'm talking about the Adonis one, not the Vite one. This is not desirable, it should be Vite that takes care of this, with HMR. 

This is because chokidar `source:change` is triggered since the tsconfig includes this `resources/js/app.ts` file

Maybe the solution would be to just ignore this folder from the tsconfig, and provide a second one that will manage this folder?

Reply:
Yup, the frontend ts file should be separate and the AdonisJS one can ignore it by default

Reply:
Not sure how we should approach that. Should we ship a new tsconfig after executing `node ace configure vite | encore` ? 
It would create a tsconfig.json in the resources folder. 

I would have preferred to have a kind of tsconfig.web.json and a tsconfig.back.json at the root but it seems complicated to automate here. The ideal would be to do it when creating fresh applications in create-adonis-app

Or should we let the users manage and create their own tsconfig for their front? What do you think?